### DISCUSSION TITLE
Can I change "Back to top" position ?

### DISCUSSION THREAD

**Discussion by SPyofgame** [squidfunk/mkdocs-material#5243] [Q&A] [answered]
Sometimes the Back to the top covers my text when reading and kinda annoying

![image](https://user-images.githubusercontent.com/62207095/226991012-fb06dd01-0bf6-4ac3-add4-d91ca8748eb9.png)

Can I somehow make it align on the top (aka the Navigation Panel) ?

**Comment by squidfunk** [comment, marked as answer]
Jup, you can change the positioning with [additional CSS](https://squidfunk.github.io/mkdocs-material/customization/#additional-css), as with all other things. Also see #2513 for a similar request.

**Reply by SPyofgame** [reply]
I inspect the button element

![image](https://user-images.githubusercontent.com/62207095/227092239-d6560463-e6cc-4358-bf56-2d8fa8b62b49.png)

I override it using extra properties

![image](https://user-images.githubusercontent.com/62207095/227092772-617a554d-0dbf-4257-aaa4-f68f1797960c.png)

It is in `docs/assets/styles/aces.css`

![image](https://user-images.githubusercontent.com/62207095/227092503-51201e53-5650-4782-8a69-6279f6a2c5d5.png)

It is called from mkdocs.yml extra_css

![image](https://user-images.githubusercontent.com/62207095/227092553-9ffc4cbf-6bb8-4fcb-914f-b11ea8d79930.png)

What i expected

![image](https://user-images.githubusercontent.com/62207095/227093170-1e315689-391d-4def-bdcf-9b41d31a693f.png)


Weird, I can't override the button

**Reply by squidfunk** [reply]
Try:

``` css
.md-top {
  z-index: 10;
  top: 5px !important; /* Edit: added afterwards, necessary in this case */
}
```

**Reply by SPyofgame** [reply]
Seems like it didnt work

![image](https://user-images.githubusercontent.com/62207095/227169854-a9170a3f-2351-4566-877b-7decf809c0ea.png)

It still not override the style for the button :<

**Reply by kamilkrzyskow** [reply]
@SPyofgame2 
I don't use this feature, but there is some kind of `top: 69px` inline CSS in that button when I looked at the reference site https://squidfunk.github.io/mkdocs-material/reference/. It's not in the plain/raw HTML, so it's added later by some JavaScript or something, but I didn't find anything specific looking over the source files.

If that is the case on your site too then you need to use the [`!important`](https://developer.mozilla.org/en-US/docs/Web/CSS/important) flag to increase the importance of your additional settings in the CSS file over the inline settings. Keep in mind that this flag is designed for such cases like this, and shouldn't be overused in the manner of "just in case I'll put it everywhere".

**Reply by SPyofgame** [reply]
Thank you, it is working. I once thought it might cause some issues hence didn't even bother trying to use it. Now I get it 👍

**Reply by squidfunk** [reply]
Jup, in this case `!important` is necessary. The reason is that while the position is set via CSS, there might be cases where authors implicitly change the size of the header, e.g. by using a bigger logo or adding additional stuff, that would now overlay the back-to-top button. The back-to-top button offset is therefore dynamically computed when JavaScript is available.

Computation is here:

https://github.com/squidfunk/mkdocs-material/blob/70d1002a1ddf0705547a01b2eeb3c874c2f804d5/src/assets/javascripts/components/top/index.ts#L160-L168

**Reply by SPyofgame** [reply]
Thanks
